README.md

```diff
@@ -10,7 +10,22 @@ First, you'll need to configure a database. Once this project is deployed, we'll
 
 ### Windows setup
 
-Coming Soon!
+## Chocolatey and Postgres (Needs Testing)
+1. [Install Choclatey](https://chocolatey.org/docs/installation#installing-chocolatey) if you do not already have it installed.
+2. Add `C:\ProgramData\chocolatey\bin` to your paths (type path in search, click Edit environment varaibles for you account).
```

Comment:
I'd maybe add a whole group related to adding values to the path. Adding to path is one of the most common issues I see with new people and python projects (and windows). Even something like a link to a url would be good.